### ISSUE TITLE
pluginslist - schema missing

### ISSUE THREAD

**Issue by KannappanSomu** [Opster/opensearch-k8s-operator#249]
@idanl21 @swoehrl-mw ;

as per #201 , Tried installing plugins by mentioning list under spec.general section for latest version of operator ( 2.02, 2.03), and all version says not supported in schema, is this a bug ? I also looked at the crd file that is applied by helm chart and plugins list field is missing, is this a bug in helm chart ? 

`OpenSearchCluster/opensearch-operator/devops-cluster dry-run failed, error: failed to create typed patch object: .spec.general.pluginsList: field not declared in schema...`

```
general:
    serviceName: devops-cluster
    version: 1.3.1
    pluginsList: ["repository-s3","https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/releases/download/1.3.0.0/prometheus-exporter-1.3.0.0.zip"]

```

**Comment by KannappanSomu** [comment]
Looks like the pluginsList field is missing from [crd template file](https://github.com/Opster/opensearch-k8s-operator/blob/main/charts/opensearch-operator/templates/opensearchclusters.opensearch.opster.io-crd.yaml) under spec.general :

**Comment by KannappanSomu** [comment]
Added PR #250  to add these fields in crd template file .
please review

**Comment by swoehrl-mw** [comment]
Fixed with PR #250
